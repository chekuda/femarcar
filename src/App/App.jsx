import React from 'react'
import classnames from 'classnames'
import image1 from 'assets/mockimage.png'

import styles from './App.module.scss'

const App = () =>
  <div
    className={classnames(styles.container, styles.text)}
    data-testid='container'
  >
    <img src={image1} />
    jose
  </div>

export default App
