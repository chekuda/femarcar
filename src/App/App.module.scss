.container {
  color: red;
  background: var(--primary-color);
  &.text {
    color: black;
  }
}