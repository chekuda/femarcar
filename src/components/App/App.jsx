import React from 'react'
import Button from '@material-ui/core/Button'
import classnames from 'classnames'

import User from '../User'
import image1 from 'assets/images/mockimage.png'

import styles from './App.module.scss'

const App = () =>
  <div
    className={classnames(styles.container, styles.text)}
    data-testid='container'
  >
    <img src={image1} />
    <User />
    <Button variant="contained" color="primary">Default</Button>
  </div>

export default App
