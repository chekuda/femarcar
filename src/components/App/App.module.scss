.container {
  display: flex;
  flex-direction: column;
  color: red;
  background: var(--primary-color);
  &.text {
    color: black;
  }
}