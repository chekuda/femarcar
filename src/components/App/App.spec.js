import React from 'react'
import { render } from '@testing-library/react'

import App from './App'

describe('given App component', () => {
  describe('when trying to render the App component', () => {
    it('should render the App component', () => {
      const { queryByTestId } = render(<App />)

      expect(queryByTestId('container')).toMatchSnapshot()
    })
  })
})
