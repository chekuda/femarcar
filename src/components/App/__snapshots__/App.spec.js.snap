// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`given App component when trying to render the App component should render the App component 1`] = `
<div
  class=""
  data-testid="container"
>
  <img
    src="[object Object]"
  />
  jose
</div>
`;
